src/ApiService/ApiService/onefuzzlib/Creds.cs

```diff
@@ -12,11 +13,21 @@ public interface ICreds
     public string GetBaseResourceGroup();
 
     public ResourceIdentifier GetResourceGroupResourceIdentifier();
+
+    public string GetInstanceName();
+
+    public Async.Task<Guid> GetInstanceId();
 }
 
 public class Creds : ICreds
 {
 
+    private IContainers _containers;
```

Comment:
don't add that dependency. it should be the other way around. Icontainer should depend on Icreds.
Icreds will eventually be a singleton

Reply:
Should I just initialize the container in the GetInstanceId directly?

Reply:
Moving GetInstanceId to Storage and out of Creds.